sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -293,15 +293,6 @@ object QueryExecutionErrors {
     new IllegalStateException("table stats must be specified.")
   }
 
-  def unaryMinusCauseOverflowError(originValue: Short): ArithmeticException = {
-    new ArithmeticException(s"- $originValue caused overflow.")
-  }
-
-  def binaryArithmeticCauseOverflowError(
-      eval1: Short, symbol: String, eval2: Short): ArithmeticException = {
-    new ArithmeticException(s"$eval1 $symbol $eval2 caused overflow.")
-  }
```

Comment:
Why did you remove these methods from this file? How about merging these funcs into a unified one like this?
```
  def arithmeticCauseOverflowError(overflowCause: String): ArithmeticException = {
    new ArithmeticException(s"$overflowCause caused overflow.")
  }
```

Reply:
> Why did you remove these methods from this file?

The PR is to align the error message of byte/short to int/long's. After the change,`binaryArithmeticCauseOverflowError` is not used in anywhere.